src/main/resources/config/liquibase/changelog/20231120095701_changelog.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="andreas (generated)" id="1700470627638-3">
```

Comment:
Please use the id that you also used for the file name here and also the "(generated)" can probably be removed. Also since you are not the only Andreas it might make sense to specify (part of) your last name here as well (or use your GitHub name), so it is directly clear who made this.
```suggestion
    <changeSet author="andreas" id="20231120095701-1">
```